tests/mixed_bag.rs

```diff
@@ -227,11 +227,32 @@ fn list_ignored() {
 #[test]
 fn list_with_filter() {
     let (c, out) = common::do_run(args(["--list", "a"]), tests());
+    // Matches all tests that contain "a" in either the name or the kind.
     assert_log!(out, "
         cat: test
+        [apple] fox: test
+        [apple] bunny: test
+        [banana] fly: test
         [banana] bear: test
         cyan: bench
         [banana] orange: bench
+        [banana] pink: bench
+    ");
+    assert_eq!(c, Conclusion {
+        num_filtered_out: 0,
+        num_passed: 0,
+        num_failed: 0,
+        num_ignored: 0,
+        num_measured: 0,
+     });
+}
+
+#[test]
+fn list_with_filter_exact() {
+    let (c, out) = common::do_run(args(["--list", "--exact", "[apple] fox"]), tests());
+    // Matches all tests that contain "a" in either the name or the kind.
```

Comment:
I think this comment is left over from copy-paste, and not accurate here...

Reply:
Addressed this, thanks.